server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10-alpine
+
+MAINTAINER eLife Reviewer Product Team <reviewer-product@elifesciences.org>
+
+WORKDIR /opt
+
+COPY package.json .
+COPY yarn.lock .
+
+COPY dist/ dist/
+
+COPY gql/ dist/modules/
+
+COPY node_modules/ ./node_modules/
+
+EXPOSE 3000
+HEALTHCHECK --interval=1m --timeout=1s \
+	CMD curl -f http://localhost:3000/health || exit 1
```

Comment:
`-f` should be enough to deliver a failed exit code?

Reply:
(without `||`)

Reply:
I think docker requires that the exit code is either `1` or `0`, I'm pretty sure `curl` will exit with `22` when there's an error code sent by the server

Reply:
:+1: seems a documented pattern to append `|| exit 1`